app/io/pathfinder/controllers/ViewController.scala

```diff
@@ -0,0 +1,9 @@
+package io.pathfinder.controllers
```

Comment:
It displays a web page when going to the server's url in the browser, we probably will end up wanting to add more to it, such as like a dashboard or something to monitor the web server. I also use it to accept migrations that play generates.